src/functions/JWT/generateJWT.js

```diff
@@ -1,6 +1,6 @@
 const jose = require("jose");
 
-module.exports = async (jti, sub, issuer, payload = {}, expiry = "1y") => {
+module.exports = async (jti, sub, issuer, payload = {}, expiry = "90d") => {
```

Comment:
@praveen-betalectic This should come from ENV, default would be 1y

Reply:
@rjvim ok will make the changes.

Reply:
@rjvim please review now, I have introduced a new env variable `JWT_EXPIRY_TIME`